DataGrid/Extension/Configuration/EventSubscriber/ConfigurationBuilder.php

```diff
@@ -19,84 +19,53 @@
 class ConfigurationBuilder implements EventSubscriberInterface
 {
     /**
-     * @var \Symfony\Component\HttpKernel\KernelInterface
+     * Datagrid configuration
+     *
+     * @var array
      */
-    protected $kernel;
+    protected $datagridConfiguration;
```

Comment:
i think we should name this variable `datagridsConfigurations` as long as it contains all datagrids' configurations